tst/EnergyPlus/unit/AirflowNetworkBalanceManager.unit.cc

```diff
@@ -8894,6 +8896,13 @@ TEST_F(EnergyPlusFixture, TestExternalNodesWithLocalAirNode)
     DataGlobals::AnyLocalEnvironmentsInModel = true;
     OutAirNodeManager::SetOutAirNodes();
     GetAirflowNetworkInput(OutputFiles::getSingleton());
+    // Issue 7656
+    std::string const error_string = delimited_string({
+        "   ** Warning ** No Timestep object found.  Number of TimeSteps in Hour defaulted to 4.",
+        "   ** Warning ** CheckUsedConstructions: There are 2 nominally unused constructions in input.",
+        "   **   ~~~   ** For explicit details on each unused construction, use Output:Diagnostics,DisplayExtraWarnings;",
+    });
+    EXPECT_TRUE(compare_err_stream(error_string, true));
```

Comment:
I was a bit confused about this unit test modification, but I think I know what happened.  The error file looks appropriate now that you added the outdoor air node into the inputs above.